docs/Changelog.md

```diff
@@ -60,6 +60,8 @@ and `GenericNodeTransformation.implementation` is not invoked in this case
 * [#2465](https://github.com/TouK/nussknacker/pull/2465) aggregate-sliding emitWhenEventLeft parameter changed default value from true to false
 * [#2474](https://github.com/TouK/nussknacker/pull/2474) Interpreter return type changed from `F[Either[List[InterpretationResult], EspExceptionInfo[_ <: Throwable]]]` to `F[List[Either[InterpretationResult, EspExceptionInfo[_ <: Throwable]]]]`.
   Hence, e.g. multiple branches in Graph can be evaluated, both positively and negatively at the same time.
+* [#2540](https://github.com/TouK/nussknacker/pull/2540) It's possible to use different Effects than `Future` in standalone engine. `InvocationMetrics` are no longer 
```

Comment:
can we stick with our normalized naming "request-response runtime" or "request-response interpreter" if is used as a librarary

Reply:
Changed, though it's 1.1, so I left a mention of 'standalone', as rename is a part of 1.2 ;)